src/test/java/pl/szczesniak/dominik/whattowatch/movies/infrastructure/persistence/InFileMoviesRepositoryIntTest.java

```diff
@@ -131,4 +131,36 @@ void next_movieid_should_be_one_higher_than_previously_created_movieid() {
 		assertThat(nextMovieId.getValue()).isEqualTo(lastMovieId.getValue() + 1);
 	}
 
+	@Test
+	void should_return_empty_when_no_matching_movieid_or_userid() {
+		// given
+		final UserId userIdOne = new UserId(1);
+		final UserId userIdTwo = new UserId(2);
+		final MovieId movieId = tut.nextMovieId();
+		final Movie movie = recreate(movieId, new MovieTitle("Parasite"), userIdOne);
```

Comment:
ten recreate miał być używany tylko do odtwarzania filmów z repo

Reply:
a nie do tworzenia sobie po prostu obiektów, do tego jest konstruktor. Skoro konstruktor jest pakietowy to można to sobie załatwić przez obiekty typu MovieSample